openshift/templates/deploy.dc.yaml

```diff
@@ -48,22 +48,23 @@ objects:
       labels:
         app: ${NAME}-${SUFFIX}
       name: ${NAME}-${SUFFIX}
-  - apiVersion: apps/v1
-    kind: Deployment
+  - apiVersion: v1
+    kind: DeploymentConfig
     metadata:
       labels:
         app: ${NAME}-${SUFFIX}
       name: ${NAME}-${SUFFIX}
     spec:
       replicas: 1
       selector:
-        matchLabels:
-          app: ${NAME}-${SUFFIX}
+        name: ${NAME}-${SUFFIX}
+      strategy:
+        type: Rolling
+      triggers: []
       template:
         metadata:
           labels:
-            app: ${NAME}-${SUFFIX}
```

Comment:
Was this removed on purpose?  app=${NAME}-${SUFFIX} is used for cleanup.

Reply:
I think I removed this because it doesn't work with DeploymentConfig.